### PR TITLE
chore(docs): Added official AWS or Amazon prefixes to all service references

### PR DESCRIPTION
**Pull request by Doug-AWS**
----

### Pull Request Checklist

* [ ] Testing
  - Unit test added
  - __CLI change?:__ coordinate update of integration tests with team
  - __cdk-init template change?:__ coordinated update of integration tests with team
* [ ] Docs
  - __jsdocs__: All public APIs documented
  - __README__: README and/or documentation topic updated
* [ ] Title and Description
  - __Change type__: title prefixed with **fix**, **feat** will appear in changelog
  - __Title__: use lower-case and doesn't end with a period
  - __Breaking?__: last paragraph: "BREAKING CHANGE: <describe what changed + link for details>"
  - __Issues__: Indicate issues fixed via: "**Fixes #xxx**" or "**Closes #xxx**"
* [ ] Sensitive Modules (requires 2 PR approvers)
  - IAM Policy Document (in @aws-cdk/aws-iam)
  - EC2 Security Groups and ACLs (in @aws-cdk/aws-ec2)
  - Grant APIs (only if not based on official documentation with a reference)

----

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license.

### CODE REVIEW

File: packages/@aws-cdk/aws-codecommit/README.md [hunk 1]
```diff
@@ -42,17 +47,17 @@ You can also add the Repository to the Pipeline directly:
 const sourceAction = repo.addToPipeline(sourceStage, 'CodeCommit');
 ```
 
-### Events
+## Events
 
-CodeCommit repositories emit CloudWatch events for certain activity.
+AWS CodeCommit repositories emit Amazon CloudWatch events for certain activity.
```
Comment (skinny85) [inline packages/@aws-cdk/aws-codecommit/README.md]: `certain activities` perhaps?

**Comment by skinny85** [comment]
I think the PR title should be `doc(aws-codecommit)`, but I'm not an expert on conventional commits...

File: packages/@aws-cdk/aws-codecommit/README.md (hunk 1 above)
Comment (Doug-AWS) [reply packages/@aws-cdk/aws-codecommit/README.md]: Good catch. I'll update the docs. BTW, I plan to ask my editor to look at the docs next week. She would have most likely caught this.

**Comment by Doug-AWS** [comment]
There shouldn't be any extra "AWS" prefixes.

**Pull request merged**